nilearn/plotting/img_plotting.py

```diff
@@ -683,26 +683,26 @@ def plot_roi(roi_img, bg_img=MNI152TEMPLATE, cut_coords=None,
     return display
 
 
-def plot_prob_atlas(maps_img, anat_img=MNI152TEMPLATE, view_type='auto',
+def plot_prob_atlas(maps_img, bg_img=MNI152TEMPLATE, view_type='auto',
                     threshold='auto', linewidths=2.5, cut_coords=None,
                     output_file=None, display_mode='ortho',
                     figure=None, axes=None, title=None, annotate=True,
                     draw_cross=True, black_bg='auto', dim='auto',
                     colorbar=False,
                     cmap=plt.cm.gist_rainbow, vmin=None, vmax=None,
-                    alpha=0.5, **kwargs):
+                    alpha=0.5, anat_img=None, **kwargs):
     """ Plot the probabilistic atlases onto the anatomical image
         by default MNI template
 
         Parameters
         ----------
         maps_img : Niimg-like object or the filename
             4D image of the probabilistic atlas maps
-        anat_img : Niimg-like object
+        bg_img : Niimg-like object
```

Comment:
Since it is not completely removed. You may need to keep the documentation of this parameter ```anat_img``` and add a NOTE: saying what you have said in warning message ?

If it sounds good. You can find reference point in fetch_haxby in nilearn/datasets/func.py

Reply:
I added it just after `bg_img`